app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<section class='main_section'>
+  <h1>Here is a list of all users</h1>
+  <ul class='users_list'>
+    <% @users.each do |user| %>
+      <li class='user_card'>
+        <div class='user_img'>
+          <img src= <%= user.photo %> alt= <%= user.name %> />
+        </div>
+
+        <div class='user_stuff'>
+          <p class='user_name'><%= user.name %></p>
```

Comment:
- [ ] The name should be a link that allows to see the user information, please 🙏🏽 refactor your code to make it possible 👇🏽 
```ruby
<p class='user_name'><%= link_to user.name, user_path(user) %></p>
```

Reply:
Done